Verify mac installation (#3760)
aiidateam/aiida-core: docs/source/install/prerequisites.rst

    brew services start postgresql

> Or, if you don't want/need a background service you can just run:

Reviewer: Thanks for the suggestion. 
Since this is something that would need to be added to installation sections for all operating systems (also ubuntu, etc.) and all services, I wonder whether there is a way to do this without bloating the instructions for all sections.

The best thing would actually be to add a section for installation via conda, where it is anyhow the user's responsibility to start/stop the services.
Just opened an issue for that https://github.com/aiidateam/aiida-core/issues/3761

.. code-block:: bash

    pg_ctl -D /usr/local/var/postgres start

For a more detailed description of database requirements and usage see the :ref:`database<database>` section.

> Installing the RabbitMQ message broke through Homebrew is as easy as:

Reviewer: Just saw this typo
```suggestion
Installing the RabbitMQ message broker through Homebrew is as easy as:
```

    brew services start rabbitmq

Or, if you don't want/need a background service you can just run:

.. code-block:: bash

    rabbitmq-server

> You can check whether it is running by checking the status through the command:

Reviewer: ```suggestion
After a few seconds, check whether it is running by checking the status through the command:
```

.. code-block:: bash

    /usr/local/sbin/rabbitmqctl status

> Wait for few seconds for running the server and then run the status:

Reviewer: ```suggestion
Wait for a few seconds until the server has started, and query its status:
```

> Wait for a few seconds until the server has started, and query its status:

Reviewer: The waiting needs to happen before you query the status I guess...
```suggestion
```
